docs/interop.md

```diff
@@ -51,18 +47,23 @@ func (t *Transaction) Claim(wallet *token.OwnerWallet, tok *token2.UnspentToken,
 func (t *Transaction) Reclaim(wallet *token.OwnerWallet, tok *token2.UnspentToken) error
 ```
 
-The interop `Wallet` service, located in `token/services/interop/wallet.go`, supports listing expired tokens, whose deadline have passed, and listing tokens with a desired matching preimage.
+The interop `Wallet` service, located under `token/services/interop/`, supports listing tokens with a desired matching preimage, and listing expired tokens, whose deadline have passed.
 
 In addition, the interop signer and verifier services are script specific, for example in the HTLC case the preimage is part of the signed message.
 
+Finally, the interoperability services which are responsible for assembling the token transaction and managing its lifecycle are the same as the [`Token Transaction Services`](./services.md).
+They are located in `token/services/interop`.
+
+
 ## Driver adjustments 
 
 The `FabToken` and `ZKAT DLog` drivers support also interoperability, and more specifically, the drivers support HTLC.
 
 In addition to the regular validation process, their `validator` ensures that in the lock and claim cases the deadline has not expired, and in the reclaim case the deadline has passed.    
-Moreover, the validator returns a `TransferAction` now holds the `ClaimPreImage`, as it is written into the ledger and can later be searched by calling the scan function located in `token/services/interop/scanner.go`
+Their `validator` also allows adding extra validation process to support more kinds of interoperability scripts.  
+Moreover, the validator returns a `TransferAction` that now holds the `ClaimPreImage`, as it is written into the ledger and can later be searched by calling the scan function located in `token/services/interop/scanner.go`
```

Comment:
the `TransferAction` carries the pre-image at time of transaction assembly. The validator just checks whether it matches the pre-image in the signature.